apps/prairielearn/src/question-servers/calculation-subprocess.js

```diff
@@ -34,22 +38,57 @@ async function prepareChunksIfNeeded(question, course) {
   await chunks.ensureChunksForCourseAsync(course.id, chunksToLoad);
 }
 
+function getQuestionRuntimePath(questionServerPath, courseHostPath, courseRuntimePath) {
+  const questionServerType = contains(courseHostPath, questionServerPath) ? 'course' : 'core';
+
+  if (questionServerType === 'course') {
+    const questionServerPathWithinCourse = path.relative(courseHostPath, questionServerPath);
+    return path.join(courseRuntimePath, questionServerPathWithinCourse);
+  }
+
+  if (config.workersExecutionMode === 'native') {
+    return questionServerPath;
+  } else {
+    const questionServerPathWithinRepo = path.relative(REPOSITORY_ROOT_PATH, questionServerPath);
+
+    // This is hardcoded to use the `/PrairieLearn` directory in our container
+    // image, which is where the repository files will be located.
+    return path.join('/PrairieLearn', questionServerPathWithinRepo);
```

Comment:
🤷 I don't love this, but it should be fine, I don't foresee us changing this path in the near future.

Reply:
Agreed.